extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -239,15 +239,19 @@ public function signup() {
 			$ret = $signupForm->addNewAccount();
 		}
 
-		$this->result = ( $signupForm->msgType == 'error' ) ? $signupForm->msgType : 'ok';
-		$this->msg = $signupForm->msg;
-		$this->errParam = $signupForm->errParam;
-
 		// pass and ID of created account for FBConnect feature
```

Comment:
I do not know this code that well. And it is difficult to catch what else might be using that. So I think I will just leave it for now.